diff --git a/arch/riscv/boot/dts/allwinner/sun20i-d1-lichee-rv-dock.dts b/arch/riscv/boot/dts/allwinner/sun20i-d1-lichee-rv-dock.dts
index 0b0f356e2..ad23d02d7 100644
--- a/arch/riscv/boot/dts/allwinner/sun20i-d1-lichee-rv-dock.dts
+++ b/arch/riscv/boot/dts/allwinner/sun20i-d1-lichee-rv-dock.dts
@@ -56,6 +56,11 @@ wifi_pwrseq: wifi-pwrseq {
 		compatible = "mmc-pwrseq-simple";
 		reset-gpios = <&gpio 6 12 GPIO_ACTIVE_LOW>; /* PG12 */
 	};
+
+	memory@40000000 {
+		device-type = "memory";
+		reg = <0x40000000 0x20000000>;
+	};
 };

 &codec {
